docs/raw/serving-community-history.md

```diff
@@ -84,200 +89,282 @@ Furthermore, it assumes that:
 - Community owner nodes receive all community messages
 - Community owner nodes are honest
 
+These assumptions are less than ideal and will be enhanced in future work. This [discussion](https://forum.vac.dev/t/status-communities-protocol-and-product-point-of-view/114) provides more information on that.
+
 ## Overview
 
-The following is a high-level overview of the user flow and features this specification describes.
+The following is a high-level overview of the user flow and features this specification describes. For more detailed descriptions, read the dedicated sections in this specification.
 
 ### Serving community history archives
 
 Community owner nodes go through the following (high level) process to provide community members with message histories (assumes community owner node is available 24/7):
 
-1. Community owner creates a Status community
-2. Community owner enables community history archive support
-3. A special type of channel for distributing magnet links ([Magnet URI scheme](https://en.wikipedia.org/wiki/Magnet_URI_scheme), [Extensions for Peers to Send Metadata Files](https://www.bittorrent.org/beps/bep_0009.html)) is created
+1. Community owner creates a Status community (previously known as [org channels](https://github.com/status-im/specs/pull/151))
+2. Community owner enables community history archive support (possibly on by default but can be turned off as well - see [UI feature spec](https://github.com/status-im/feature-specs/pull/36))
```

Comment:
Remove word "possibly", target is on by default

Reply:
Sounds good!